### DISCUSSION TITLE
[Suggestion] Removing some legacy transparent menus

### DISCUSSION THREAD

**Discussion by telk5093** [OpenTTD/OpenTTD#9185] [Features]
We already have **Transparency Options** window to show/hide houses in town and signs.
There is no reason to keep these two menu in option menu IMO. Why don't we remove these legacy menu?

![image](https://user-images.githubusercontent.com/10726524/116902099-b2738b80-ac75-11eb-90ac-eb6ab87ea103.png)

**Comment by unbeatable-101** [comment]
It's a way to quickly access them, I for one use the `transparent buildings` option in there all the time
